docs/asch_sdk_api_en.md

```diff
@@ -0,0 +1,622 @@
+        - [1.10 app.sdb.del(model, cond)](#110-appsdbdelmodel-cond)
+    - [2. Balance](#2-balance)
+        - [2.1 app.balances.get(address, currency)](#21-appbalancesgetaddress-currency)
+        - [2.2 app.balances.increase(address, currency, amount)](#22-appbalancesincreaseaddress-currency-amount)
+        - [2.3 app.balances.decrease(address, currency, amount)](#23-appbalancesdecreaseaddress-currency-amount)
+        - [2.4 app.balances.transfer(currency, amount, from, to)](#24-appbalancestransfercurrency-amount-from-to)
+    - [3 Data model](#3-data-model)
+        - [3.1 app.model[name]](#31-appmodelname)
+        - [3.2 fields()](#32-fields)
+        - [3.3 count(cond)](#33-countcond)
+        - [3.4 exists(cond)](#34-existscond)
+        - [3.5 findOne(options)](#35-findoneoptions)
+        - [3.6 findAll(options)](#36-findalloptions)
+    - [4. Routing](#4-routing)
+        - [4.1 app.route.get(path, handler)](#41-approutegetpath-handler)
+        - [4.2 app.route.post(path, handler)](#42-approutepostpath-handler)
+        - [4.3 app.route.put(path, handler)](#43-approuteputpath-handler)
+    - [5. Cost pool](#5-cost-pool)
+        - [5.1 app.feePool.add(currency, amount)](#51-appfeepooladdcurrency-amount)
+    - [6. Icrement ID](#6-icrement-id)
+        - [6.1 app.autoID.get(name)](#61-appautoidgetname)
+        - [6.2 app.autoID.increment(name)](#62-appautoidincrementname)
+    - [7. 日志](#7-日志)
+        - [7.1 app.logger.setLevel(level)](#71-apploggersetlevellevel)
+        - [7.2 app.logger.log()](#72-apploggerlog)
+        - [7.3 app.logger.trace()](#73-apploggertrace)
+        - [7.4 app.logger.debug()](#74-apploggerdebug)
+        - [7.5 app.logger.info()](#75-apploggerinfo)
+        - [7.6 app.logger.warn()](#76-apploggerwarn)
+        - [7.7 app.logger.error()](#77-apploggererror)
+    - [8. Tools](#8-tools)
+        - [8.1 app.validate(type, value)](#81-appvalidatetype-value)
+        - [8.2 app.registerContract(type, name)](#82-appregistercontracttype-name)
+        - [8.3 app.getContractName(type)](#83-appgetcontractnametype)
+        - [8.4 app.registerFee(type, min, currency)](#84-appregisterfeetype-min-currency)
+        - [8.5 app.getFee(type)](#85-appgetfeetype)
+        - [8.6 app.setDefaultFee(min, currency)](#86-appsetdefaultfeemin-currency)
+        - [8.7 app.getRealTime(epochTime)](#87-appgetrealtimeepochtime)
+        - [8.8 app.registerHook](#88-appregisterhook)
+        - [8.9 app.custom[]](#89-appcustom)
+
+Table created with [Markdown-TOC](https://github.com/AlanWalk/Markdown-TOC).
+
+
+# Asch SDK API
+
+
+## **1. Database**
+
+### 1.1 `aync` app.sdb.load(model, fields, indices)
+
+- `model` Datamodel
+- `fields` Fields of the data model
+- `indices` Array of indexes, it can be a single field index or a multi field index (provide an string array)
+
+> The operation has no return value, it throws an Exception when an error occurs
+> 将指定模型的数据加载到内存并建立索引, 这样可以提高查询和更新一个状态的效率
```

Comment:
why Chinese here?

Reply:
I couldn't translate it in a meaningful way.